proteuslib/unit_models/tests/test_reverse_osmosis_0D.py

```diff
@@ -649,23 +655,37 @@ def test_Pdrop_calculation(self):
         assert results.solver.status == SolverStatus.ok
 
         # test solution
-        assert (pytest.approx(-9.173e5, rel=1e-3) == value(m.fs.unit.deltaP[0]))
-        assert (pytest.approx(1.904e-3, rel=1e-3) ==
+        assert (pytest.approx(151.622, rel=1e-3) == value(m.fs.unit.N_Re_io[0, 'in']))
+        assert (pytest.approx(0.2089, rel=1e-3) == value(m.fs.unit.velocity_io[0, 'in']))
+        assert (pytest.approx(63.001, rel=1e-3) == value(m.fs.unit.N_Re_io[0, 'out']))
+        assert (pytest.approx(0.0914, rel=1e-3) == value(m.fs.unit.velocity_io[0, 'out']))
+        assert (pytest.approx(151.622, rel=1e-3) == value(m.fs.unit.velocity_io[0, 'in']
+                * m.fs.unit.feed_side.properties_in[0].dens_mass_phase['Liq']
+                * m.fs.unit.dh
+                / m.fs.unit.feed_side.properties_in[0].visc_d_phase['Liq']))
+        assert (pytest.approx(63.001, rel=1e-3) == value(m.fs.unit.velocity_io[0, 'out']
+                * m.fs.unit.feed_side.properties_out[0].dens_mass_phase['Liq']
+                * m.fs.unit.dh
+                / m.fs.unit.feed_side.properties_out[0].visc_d_phase['Liq']))
+        assert (pytest.approx(-2.786e5, rel=1e-3) == value(m.fs.unit.deltaP[0]))
```

Comment:
This is a much higher pressure drop than I'd expect given my OARO paper. The average Reynolds number is ~90, but it has about 35kPa/m pressure drop. Nearly 10x more than the OARO paper, which was operating at a much higher Reynolds number ~600 .

Reply:
We know that there was a mistake in the hydraulic diameter calculation of the OARO paper, but it shouldn't make this big of a difference. You can compare some of these results to my membrane module modeling paper https://www.sciencedirect.com/science/article/pii/S0376738818326115, which has the corrected hydraulic diameter calculation and uses a similar porosity to your test case.

Reply:
I would think that any major differences would be attributed to the selection of (not limited to) feed mass flow rate, membrane dimensions, and spacer geometry, and I don't think I matched up all specified variables with your original study in this particular test. 

I changed all fixed variables to match your study from the link you provided, and it seems like my results match up although I didn't see what your pressure drop result was. 

The results I get for pressure drop and dP/dx are 1.66 MPa and 0.104 MPa.

Additionally, although the solver gets a feasible solution, I get some badly scaled variables  when using the specifications in your study (had to disable assert to check results). Maybe we can discuss.